src/upload.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
 set -e
-mc alias set remote "$STORAGE_ENDPOINT" "$STORAGE_ACCESS_KEY" "$STORAGE_SECRET_KEY" --api "$STORAGE_SIGNATURE_VERSION"
-mc cp --recursive /backups/output/ remote/"$STORAGE_BUCKET_NAME"
+AWS_ACCESS_KEY_ID=$STORAGE_ACCESS_KEY \
```

Comment:
This is here so that:
a. we don't need chart changes
b. people could get confused about how we now need AWS credentials